src/pages/RequestCallPage.js

```diff
@@ -84,6 +89,12 @@ const propTypes = {
 
     /** Information about the network from Onyx */
     network: networkPropTypes.isRequired,
+
+    /** Holds information about the users account that is requesting the call */
+    account: PropTypes.shape({
+        /** Success state to show confirmation of requesting a call */
+        success: PropTypes.boolean,
```

Comment:
Would it be better to use `requestCallForm` to store this? `account` is pretty generic. 

`account.success` doesn't really scream  "Success state to show confirmation of requesting a call"

Maybe we can err on the side of being specific here and make it `requestCallForm.didRequestCallSucceed` or something?